src/NuGet.Core/NuGet.Common/Telemetry/ITelemetrySession.cs

```diff
@@ -1,8 +1,11 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-namespace NuGet.VisualStudio.Facade.Telemetry
+namespace NuGet.Common
```

Comment:
Earlier it was part of VisualStudio facade project in client sln but now i've moved into common project in core sln. And the reason to move it there is to have more flexibility and ability to emit telemetry events across solutions.